src/ebonite/core/objects/wrapper.py

```diff
@@ -30,8 +30,7 @@ class ModelWrapper(EboniteParams):
     """
     type = None
 
-    def __init__(self):
-        self.model = None
+    model = None
```

Comment:
Could you please describe why do we need this change?

Reply:
Got some strange behavior when CallableMethodModelWrapper has no attribute .model, that means this constructor was never called. Its strange because CallableMethodModelWrapper doent have it's own constructor, as it's base PickleModelWrapper. This was a quick fix, but i'll investigate this further

Reply:
it led me to a deepcopy-ing bug, it is fixed now!